winter/schema/type_inspection.py

```diff
@@ -183,7 +184,8 @@ def inspect_dataclass(hint_class) -> TypeInfo:
     return TypeInfo(type_=openapi.TYPE_OBJECT, properties=properties)
 
 
-@register_type_inspector(object, checker=lambda instance: getattr(instance, '__supertype__', None) is not None)
+@register_type_inspector(types.FunctionType,
+                         checker=lambda instance: getattr(instance, '__supertype__', None) is not None)
```

Comment:
Why test is not changed?

Reply:
Because FunctionType inherits from object and I cannot find a test which was fail before this changes

Reply:
Then what it fixes?

Reply:
Perfomance
Before when you call inspect_type(some_dataclass), inspect_new_type could be called (and could not) because dataclass is object and before this changes inspect_new_type handle object
but it is better to handle FunctionType

Reply:
Now I can write test